script/hassfest/manifest.py

```diff
@@ -53,6 +55,7 @@ def verify_uppercase(value: str):
     {
         vol.Required("domain"): str,
         vol.Required("name"): str,
+        vol.Optional("version"): str,
```

Comment:
We should validate it with `awesomeversion` that it's a valid version.

Reply:
We do that in the function, that way it does not try for core integrations

Reply:
I see your point. I guess that's valid. 

I guess the idea would be that core integrations don't have a `version` field.